tests/integration/test_create_token.py

```diff
@@ -0,0 +1,134 @@
+from app.authentication.authenticator import decrypt_token
+from tests.integration.app_context_test_case import AppContextTestCase
+from tests.integration.create_token import (
+    PAYLOAD_V2_BUSINESS,
+    PAYLOAD_V2_SOCIAL,
+    PAYLOAD_V2_SUPPLEMENTARY_DATA,
+)
+from tests.integration.integration_test_case import IntegrationTestCase
+
+
+class TestCreateToken(IntegrationTestCase, AppContextTestCase):
+    """
+    The purpose of this test class is to test the creation of a token (from create_token.py) to ensure
+    metadata in a decrypted token is nested/found in the correct level.
+    """
+
+    def test_payload_from_token(self):
+        test_parameters = [
+            {
+                "schema": "test_metadata_routing.json",
+                "theme": "default",
+                "additional_payload": {
+                    "flag_1": 123,
+                    "period_id": "abc",
+                    "link": "https://example.com",
+                },
+                "payload": PAYLOAD_V2_BUSINESS,
+            },
+            {
+                "schema": "social_demo.json",
+                "theme": "social",
+                "additional_payload": {
+                    "flag_1": True,
+                    "user_id": "abc",
+                    "date": "2016-05-12",
+                },
+                "payload": PAYLOAD_V2_SOCIAL,
+            },
+        ]
+        for value in test_parameters:
+            with self.subTest():
+                additional_payload = value.get("additional_payload")
```

Comment:
The .gets in this file can just be bracket access as we know the values exist in the dict

Reply:
Updated the ones for the test values where we know the values exist, didn't do the same for `decrypted_token`, I wasn't sure if those `.get`'s should be changed?

Reply:
Yeah even those ones can be updated as we expect them to exist as part of the test 👍

Reply:
Updated 👍